src/main/java/bgu/spl/mics/application/passiveObjects/Inventory.java

```diff
@@ -23,13 +31,14 @@
      * @post return != null
      */
     public static Inventory getInstance() {
-        //TODO: Implement this
-        return null;
+        if (_instance == null)
+            _instance = new Inventory();
+        return _instance;
```

Comment:
Maybe add a comment why Inventory doesn't have to be thread safe

Reply:
it is written in the class constructor